docs/rancher/cloud-provider.md

```diff
@@ -202,31 +212,19 @@ After deploying the `Harvester Cloud provider`, you can use the Kubernetes `Load
   
 
 ### IPAM
-Harvester's built-in load balancer supports both `pool` and `dhcp` modes. You can select the mode in the Rancher UI. Harvester adds the annotation `cloudprovider.harvesterhci.io/ipam` to the service behind.
+Harvester's built-in load balancer supports both **DHCP** and **Pool** modes, and you can select the mode in the Rancher UI. Harvester adds the annotation `cloudprovider.harvesterhci.io/ipam` to the service. Additionally, Harvester cloud provider provides a special **Share IP** mode where a service will share its load balancer IP with other services.
 
-- pool: You should configure an IP address pool in Harvester's `Settings` in advance. The Harvester LoadBalancer controller will allocate an IP address from the IP address pool for the load balancer.
-  
-  ![](/img/v1.2/rancher/vip-pool.png) 
-  
-- dhcp:  A DHCP server is required. The Harvester LoadBalancer controller will request an IP address from the DHCP server.
+- **DCHP: ** A DHCP server is required. The Harvester load balancer controller will request an IP address from the DHCP server.
+
+- **Pool:** You need an IP pool configured in the Harvester UI. The Harvester load balancer controller will allocate an IP for the load balancer service following [the IP pool selection policy](/networking/ippool.md/#selection-policy).
+
+- **Share IP:** When creating a new load balancer service, you can select an existing load balancer service to get its load balancer IP. The new service is called the secondary service, and the currently chosen service is called the primary service. You can specify the primary service in the secondary service by annotation `cloudprovider.harvesterhci.io/primary-service`. There are two limitations:
+  - It's not allowed for the secondary service to share the IP with other services.
+  - All the services sharing the same IP can not have duplicated ports.
 
 :::note
 
 It is not allowed to modify the IPAM mode. You need to create a new service if you want to modify the IPAM mode.
 
 :::
 
-### Health Checks
```

Comment:
prefer to add a note that `Health Checks` is not supported in v1.2.0 instead of just removing it directly.

Reply:
I add a note.

Reply:
Add more explanation about Health Checks.